Source/Core/Common/DirectoryIterator.cpp

```diff
@@ -0,0 +1,242 @@
+// This file is under the public domain.
+
+#include "Common/DirectoryIterator.h"
+
+#include <string>
+#include <utility>
+
+#ifdef _WIN32
+#include <Windows.h>
+#else
+#include <dirent.h>
+#include "Common/FileUtil.h"
+#endif
+
+#include "Common/StringUtil.h"
+
+namespace Common
+{
+DirectoryEntry::DirectoryEntry()
+{

```

Comment:
This can likely be ` = default;` instead.

Reply:
Done.